src/v0/destinations/mautic/transform.js

```diff
@@ -154,20 +153,29 @@ const identifyResponseBuilder = async (message, Config, endpoint) => {
 
 const process = async (event) => {
   const { message, destination } = event;
-  const { password, subDomainName, userName } = destination.Config;
-  const endpoint = `${BASE_URL.replace('subDomainName', subDomainName)}`;
+  const { password, subDomainName, domainName, domainMethod } = destination.Config;
```

Comment:
Why are we removing support of userName?

Reply:
For self hosted we are not sure what username would be like.

Reply:
But it is being used in basicAuth. Isn't it?